chromium/background.js

```diff
@@ -118,32 +119,40 @@ chrome.webNavigation.onCompleted.addListener(function() {
   updateState();
 });
 
+var USER_RULE_KEY = 'userRules';
+// Records which tabId's are active in the HTTPS Switch Planner (see
+// devtools-panel.js).
+var switchPlannerEnabledFor = {};
+// Detailed information recorded when the HTTPS Switch Planner is active.
+// Structure is:
+//   switchPlannerInfo[tabId]["rw"/"nrw"][resource_host][active_content][url];
+// rw / nrw stand for "rewritten" versus "not rewritten"
+var switchPlannerInfo = {};
+
 /**
 * Load stored user rules
  **/
-var getStoredUserRules = function() {
+function getStoredUserRules() {
   var oldUserRuleString = ls.getItem(USER_RULE_KEY);
   var oldUserRules = [];
   if (oldUserRuleString) {
     oldUserRules = JSON.parse(oldUserRuleString);
   }
   return oldUserRules;
-};
+}
 var wr = chrome.webRequest;
```

Comment:
If it isn't a big deal, can we  remove this, or move it to the top of the file? I think alias' are fine, but it is nice to have them declared upfront so we know to look for them.